### PR TITLE
Minor changes in README.zh.md.

### PR DESCRIPTION
**Pull request by inVains**
I have squashed some commits, and may have conflicts with my previous pr/merge.
If something goes wrong please tell me.
Sorry for the inconvenience. I'm struggling with git tool.

### CODE REVIEW

**Comment by digininja** [comment]
GitHub is saying there are conflicts and I can't resolve them as I don't know which is right.

And I think everyone struggles with git!

**Comment by inVains** [comment]
Ok, I'm working on it.

**Comment by inVains** [comment]
Finally done... Please check it again.
Thanks.

**Comment by digininja** [comment]
I'll accept it if you are happy with it.

File: README.zh.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Translation
 
-翻译：@[inVains](https://github.com/inVains)
+翻译：@[inVains](https://github.com/inVains) @[songzy12](https://github.com/songzy12)
```
Comment (songzy12) [inline README.zh.md:3]: How about just add me as "校对"? 

I think you take the major effort for translation and I only spent some time to review/check it.

Comment (inVains) [reply README.zh.md:3]: You're definitly worth it and thanks for you great help.

**Comment by inVains** [comment]
Sorry for the messy commits.
I'm just not sure how many commits to be squashed (git rebase -i HEAD~?) will cause no conflicts and be clean.
Or I just leave it, it can be squashed when Pull Request is accepted/closed?
image

**Comment by digininja** [comment]
I have no idea, I've never squashed anything and don't know why you would
do it so can't advise.

If you just get a PR to the point you are happy with it then I assume I can
just accept it but that may just be wishful thinking.

**Comment by songzy12** [comment]
That is because I suggested this to keep git history clean lol.

Hold on a few minutes I am trying to write a step-by-step instruction for how to do this.

**Comment by songzy12** [comment]
The overall idea would be to first rebase the commits in current repo to upstream HEAD, then do normal rebase (squash) for the already rebased commits. Appendix has a full command history I have run for reference. Here are steps with explanations.

1. (optional) Check how the current status looks like, I am using the tool `tig`:

dde2b8c8e4fc60d2f66a6c4a3887da8

We can see there are quite some branches on the top.

2. Add the author's repo as upstream, and fetch (rather than merge) the master branch:

```
git remote add upstream https://github.com/digininja/DVWA.git
git fetch upstream master
```

For now we have a branch `upstream/master` to refer to.

3. Then we rebase the changes in our current repo to `upstream/master`:

```
git rebase upstream/master
```

A conflict would happen now (since git cannot do the auto merge), you should resolve it manually by first editing `README.zh.md` to resolve the lines between `<<<` or `>>>`:

397b832140661024beec7b65d808181

Search for `<<<` to make sure there are no more conflicts to resolve, then continue the rebase

```
git add README.zh.md
git rebase --continue
```

A second conflict happend (since git apply commits one by one and there are multiple commits), we just repeat the above step, first resolve conflicts by editing `README.zh.md`, then continue  the rebase:

```
git add README.zh.md
git rebase --continue
```

In our case, the rebase would succeed now.

4. (optional) Check what the git history looks like now by using `tig`:

65566e8a0aa84a9eefc48ea5a68cdc2

You can see now we have three clean commits on top of the master HEAD.

5. Then we just need to squash the last 3 commits into 1 by another rebase to `HEAD~3`:

```
 git rebase -i HEAD~3
```

541437af13096dea8fecacd6788c208
f5bdbe3e3aa2123bdce957c45d69eb0

6. (optional) Check the git history now:

fda8249edad9196a1d23816af581e69

7. Now you can do a force push, the PR would contain one clean commit.

Cheers.

Appendix is the full history of the commands I have run:
```
➜  /tmp git clone https://github.com/inVains/DVWA.git
Cloning into 'DVWA'...
remote: Enumerating objects: 3411, done.
remote: Counting objects: 100% (98/98), done.
remote: Compressing objects: 100% (63/63), done.
remote: Total 3411 (delta 48), reused 66 (delta 33), pack-reused 3313
Receiving objects: 100% (3411/3411), 1.65 MiB | 3.74 MiB/s, done.
Resolving deltas: 100% (1518/1518), done.
➜  /tmp cd DVWA
➜  DVWA git:(master) tig
➜  DVWA git:(master) git remote add upstream https://github.com/digininja/DVWA.git
➜  DVWA git:(master) git fetch upstream master
From https://github.com/digininja/DVWA
 * branch            master     -> FETCH_HEAD
 * [new branch]      master     -> upstream/master
➜  DVWA git:(master) git rebase upstream/master
First, rewinding head to replay your work on top of it...
Applying: Internationalisation section added in READMEs and Translation section added in README.zh.md.
Using index info to reconstruct a base tree...
M       README.md
M       README.zh.md
.git/rebase-apply/patch:34: trailing whitespace.
该文件有多种语言版本：
warning: 1 line adds whitespace errors.
Falling back to patching base and 3-way merge...
Auto-merging README.zh.md
CONFLICT (content): Merge conflict in README.zh.md
error: Failed to merge in the changes.
Patch failed at 0001 Internationalisation section added in READMEs and Translation section added in README.zh.md.
hint: Use 'git am --show-current-patch' to see the failed patch
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
➜  DVWA git:(285307b) ✗ vi README.zh.md
➜  DVWA git:(285307b) ✗ git add README.zh.md
➜  DVWA git:(285307b) ✗ git rebase --continue
Applying: Internationalisation section added in READMEs and Translation section added in README.zh.md.
Applying: Minor changes in README.zh.md.
Applying: format fixed.
Using index info to reconstruct a base tree...
M       README.zh.md
Falling back to patching base and 3-way merge...
Auto-merging README.zh.md
CONFLICT (content): Merge conflict in README.zh.md
error: Failed to merge in the changes.
Patch failed at 0003 format fixed.
hint: Use 'git am --show-current-patch' to see the failed patch
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
➜  DVWA git:(ca24ec8) ✗ vi README.zh.md
➜  DVWA git:(ca24ec8) ✗ git add README.zh.md
➜  DVWA git:(ca24ec8) ✗ git rebase --continue
Applying: format fixed.
➜  DVWA git:(master) tig
➜  DVWA git:(master) git rebase -i HEAD~3
[detached HEAD 3b7e235] Minor changes in README.zh.md.
 Author: qinjin <2282229+inVains@users.noreply.github.com>
 Date: Wed Jun 30 19:54:22 2021 +0800
 1 file changed, 3 insertions(+), 5 deletions(-)
Successfully rebased and updated refs/heads/master.
➜  DVWA git:(master) tig
```

**Comment by digininja** [comment]
That's better than my usual approach:

Copy off the files I want
Delete the fork/repo
Create new fork/repo
Put files back in.

**Comment by songzy12** [comment]
lol that is indeed much easier!

**Comment by inVains** [comment]
Done.

Sorry for the late. I'm on working today.
@songzy12 
And.. oops, what a helpful instructions.
After reviewing the basis of git command and your explanations, I begin to understand it, sort of... :p

@digininja
Next time I'll try your straightforward approach. :p

Thanks for you two.

**Comment by digininja** [comment]
Is this ready to be merged in now?

**Pull request merged**